data/dark_energy_source_1.json

```diff
@@ -9,8 +9,7 @@
             "shortname": "DM1",
             "reference": "Rubin80",
             "other_references": "Perlmutter99",
-            "comments": "This is a cool dark matter source!",
-            "regimes": "optical"
```

Comment:
The Sources table does not have regimes. Also, the column name elsewhere is regime without the s.